sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v2/query.sql

```diff
@@ -59,14 +59,34 @@ metric_ping_clients_feature_usage AS (
     SUM(COALESCE(metrics.counter.addresses_updated, 0)) AS addresses_modified,
     MAX(COALESCE(metrics.quantity.addresses_saved_all, 0)) AS currently_stored_addresses,
     --Bookmark
-    SUM(COALESCE(metrics_bookmarks_add_table.value, 0)) AS bookmarks_add,
-    SUM(COALESCE(metrics_bookmarks_delete_table.value, 0)) AS bookmarks_delete,
-    SUM(COALESCE(metrics_bookmarks_edit_table.value, 0)) AS bookmarks_edit,
-    SUM(COALESCE(metrics_bookmarks_open_table.value, 0)) AS bookmarks_open,
-    MAX(
+    SUM(
```

Comment:
@bani does the v2 now also need to be backfilled with historical data?

Reply:
@kik-kik as it is, Katie already backfilled and the data is up to date / currently running in the DAG.
The code change I made in this PR is more to make it more idiomatic an future proof if the number of keys change significantly, but the numbers don't change meaningfully to require another backfill.
That said, I've identified another issue in the code so I may need to cancel this PR and remove the v2 table. That's why it's still draft.